pexpect/pxssh.py

```diff
@@ -195,7 +208,10 @@ def sync_original_prompt (self, sync_multiplier=1.0):
         can take 12 seconds. Low latency connections are more likely to fail
         with a low sync_multiplier. Best case sync time gets worse with a
         high sync multiplier (500 ms with default). '''
-
+        
+        if self.disable_original_sync:
+            return(True)
```

Comment:
Also, rather than making the method a no-op, let's check it in `login()` and skip calling the method.